checks.d/docker_daemon.py

```diff
@@ -575,11 +575,14 @@ def _report_performance_metrics(self, containers_by_id):
 
             tags = self._get_tags(container, PERFORMANCE)
 
-            self._report_cgroup_metrics(container, tags)
-            if "_proc_root" not in container:
-                containers_without_proc_root.append(DockerUtil.container_name_extractor(container)[0])
-                continue
-            self._report_net_metrics(container, tags)
+            try:
+                self._report_cgroup_metrics(container, tags)
+                if "_proc_root" not in container:
+                    containers_without_proc_root.append(DockerUtil.container_name_extractor(container)[0])
+                    continue
+                self._report_net_metrics(container, tags)
+            except BogusPIDException:
+                self.log.exception('Unable to report cgroup metrics.')
```

Comment:
This is still going to fill the logs with tracebacks. Maybe `log.error` is enough? With the container id or name for reference.